README.md

```diff
@@ -108,6 +108,7 @@ A curated list of awesome companies that offer their tools and services for free
 
 ## Miscellaneous
 
+- [Algolia](https://www.algolia.com) - Search as a service API
```

Comment:
Thanks for the submission! I had no idea Algolia offered an OSS tier! While they do have a free tier, their Open Source specific one does require approval, so I think a `requires-approval` tag is appropriate here.